packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -147,7 +150,18 @@ describe('sqlite', () => {
     expect(ctx.mocked['console.error'].mock.calls).toMatchSnapshot()
   })
 
-  it('first migration (prompt)', async () => {
+  //
+  // TODO: Windows: test fails with:
+  //
+  // [Error: Failed to create a new migration directory.
+  //    0: migration_core::api::CreateMigration
+  //            with migration_name="xl556ba8iva0gd2qfoyk2fvifsysnq7c766sscsa18rwolofgwo6j1mwc4d5xhgmkfumr8ktberb1y177de7uxcd6v7l44b6fkhlwycl70lrxw0u7h6bdpuf595n046bp9ek87dk59o0nlruto403n7esdq6wgm3o5w425i7svaw557latsslakyjifkd1p21jwj1end" draft=false
+  //              at migration-engine\core\src\api.rs:94
+  // ]
+  //
+  // Probably the file name is too long for Windows?
```

Comment:
That could become a fun bug report - solution is probably to make sure we truncate or make sure string is shorter.